playbook/app/pb_kits/playbook/pb_card/_card_mixin.scss

```diff
@@ -28,7 +28,8 @@ $pb_card_header_colors: map-merge(map-merge($product_colors, $additional_colors)
 
 @mixin pb_card_selected($border_color: $primary) {
   border-color: $border_color;
-  border-width: $pb_card_border_width * 2;
+  border-width: $pb_card_border_width;
```

Comment:
changing the border size made the card "jump" so i gave it a blue outline to maintain 2 pixels of blueness when selected